src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+    public static void main(String[] args) throws DukeException, IOException {
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+
+        String line = null;
+        String formattedLine;
+        String[] newline;
+
+        try {
+            System.out.println("Here's your content from last time!");
+            printFileContents(file);
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found");
+        }
+
+        while (!Objects.equals(line, "bye")) {
+            line = in.nextLine();
+            formattedLine = line.substring(1); //chops off first letter
+
+            String[] arr = line.split(" ", 2);
+            if (Objects.equals(line, "list")) {
+                System.out.println("Here are the tasks in your list:");
+                printTask();
+            }
+            else if (Objects.equals(arr[0], "remove")) {
+                System.out.println("Removing to do list item boss!");
+                remove(Integer.parseInt(arr[1]));
+            }
+            else if (Objects.equals(arr[0], "done")) {
+                System.out.println("Nice! I've marked Task " + arr[1] + " as done!");
+                markDone(Integer.parseInt(arr[1]));
+            }
+            else if (Objects.equals(arr[0], "d")) {
+//                try {
+//                    if (formattedLine.contains("/by")) {
+                        newline = formattedLine.split("/by");
+                        Deadline t = new Deadline(newline[0], newline[1]);
+                        addTask(t, 1);
+                        System.out.println("Gotcha! I've added this deadline");
+                        saveFile(i);
+//                    }
+//                } catch (DukeException e) {
+//                    System.out.println("OOPS! The description cannot be empty. Please input again!");
+//                }
+            }
+            else if (Objects.equals(arr[0], "e")) {
+//                try {
+//                    if (formattedLine.contains("/at")) {
+                        newline = formattedLine.split("/at");
+                        Event t = new Event(newline[0], newline[1]);
+                        addTask(t, 2);
+                        System.out.println("Gotcha! I've added this event");
+                        saveFile(i);
+//                    }
+//                } catch (DukeException e) {
+//                    System.out.println("OOPS! The description cannot be empty. Please input again!");
+//                }
+            }
+            else if (Objects.equals(arr[0], "t")){
+                try {
+                    Task t = new Task(formattedLine);
+                    addTask(t, 3);
+                    System.out.println("Gotcha! I've added this todo");
+                    saveFile(i);
+                    if (!Objects.equals(line, "bye")) System.out.println("added: " + formattedLine);
```

Comment:
The body should be wrapped by brackets irrespective of how many lines there are in the body. In other words, the conditional should be put on a separate line and wrapped in curly brackets.